packages/vue-renderer/src/renderer.js

```diff
@@ -408,7 +408,11 @@ export default class VueRenderer {
 
     // Calculate CSP hashes
     const cspScriptSrcHashes = []
-    if (this.context.options.render.csp) {
+    const csp = this.context.options.render.csp
+    const containsUnsafeInlineScriptSrc = csp && csp.policies && csp.policies['script-src'] && csp.policies['script-src'].includes(`'unsafe-inline'`)
```

Comment:
Maybe even 

```suggestion
    const containsUnsafeInlineScriptSrc = csp.policies && csp.policies['script-src'] && csp.policies['script-src'].includes(`'unsafe-inline'`)
```

as the csp truth check is already done before

Reply:
I think this is still necessary -- if `csp` were undefined but we tried to access `policies` we'd get a "Cannot read property..." error

Reply:
Whoops, you are right!